### PR TITLE
feat: initial desktop starting environment

### PR DESCRIPTION
**Pull request by NektariosFifes**
**Description**

- Basic electron application that only has a welcome screen. 
- A test ipc command between main process and renderer.

**Related issue(s)**
Progresses#94

### CODE REVIEW

**Review by jonaslagoni — COMMENTED** [review]
Awesome 👍 

You have committed a submodule `desktop`, was that on purpose or?

**Comment by NektariosFifes** [comment]
i dont know why that is present. I thought i deleted it but it persists

**Comment by NektariosFifes** [comment]
@jonaslagoni  ok removed it

**Review by jonaslagoni — APPROVED** [review]
Keep in mind that with no documentation or tests it's hard for users to use it 🙂

File: package.json [hunk 1]
```diff
@@ -38,7 +39,8 @@
     "commander": "^7.2.0",
     "cross-env": "^7.0.3",
     "js-yaml": "^3.14.1",
-    "randexp": "^0.5.3"
+    "randexp": "^0.5.3",
+    "run-script": "^0.1.1"
```
Comment (jonaslagoni) [inline package.json:43]: @NektariosFifes not sure why you use this. But be careful when using dependencies with such a low activitiy.

Easy to introduce vulnerabilities

**Pull request merged**